public/components/register_model/__tests__/setup.tsx

```diff
@@ -29,10 +34,15 @@ export async function setup(options?: RenderWithRouteProps) {
     data: [],
     pagination: { totalRecords: 0, currentPage: 1, pageSize: 1, totalPages: 0 },
   });
+
   // fill model name
-  await user.type(nameInput, 'test model name');
+  if (!options?.ignoreFillFields?.includes('name')) {
```

Comment:
what do you think if we do this instead of introducing an extra function parameter?
```typescript
  const nameInput = screen.queryByLabelText<HTMLInputElement>(/^name$/i);
  if (nameInput) {
    await user.type(nameInput, 'test model name');
  }
```

Reply:
This is not a same case, we need this option in import pre-trained model. The name and description input is exists in import model form. It's hard to check if name and description prefilled without this option.

Reply:
That's right, I see the point now